client/my-sites/plans/main.jsx

```diff
@@ -69,11 +119,10 @@ const Plans = React.createClass( {
 } );
 
 export default connect(
-	( state, props ) => {
-		return {
-			plans: getPlans( state ),
-			sitePlans: getPlansBySite( state, props.sites.getSelectedSite() ),
-			siteId: getSelectedSiteId( state )
-		};
-	}
+	( state ) => ( {
+		plans: getPlans( state ),
+		sitePlans: getPlansBySite( state, getSelectedSiteId( state ) ),
```

Comment:
Did you mean `getPlansBySiteId`? We can also store selectedSiteId as a local var.

```
( state ) => {
    const selectedSiteId = getSelectedSiteId( state );
    return {
       //...
    };
}
```